spec/helpers/friends_helper_spec.rb

```diff
@@ -15,7 +13,7 @@
 
     it 'returns the last checkin address if it exists' do
       expect(helper.friends_device_last_checkin(device.checkins.first)).to match('Last available')
-      expect(helper.friends_device_last_checkin(device.checkins.first)).to match(device.checkins.first.fogged_city.to_s)
+      expect(helper.friends_device_last_checkin(device.checkins.first)).to match(device.checkins.first.city.to_s)
```

Comment:
Depends on the checkin. The checkin city will sometimes have been set to 'fogged_city' if the device is fogged and the friend can not bypass fogging. However usually the fogged_city == city anyway. I think in this test though the device is fogged so hence why we use fogged_city.